plugins/command-manager/scripts/populate-agents-index.sh

```diff
@@ -0,0 +1,197 @@
+    "cpu": {"usage": $(echo "scale=2; $RANDOM % 100" | bc)},
+    "disk": {"read": {"bytes": $((RANDOM % 1000001))}, "write": {"bytes": $((RANDOM % 1000001))}},
+    "domain": "domain$((RANDOM % 1000))",
+    "geo": {
+      "city_name": "$(shuf -e 'San Francisco' 'New York' Berlin Tokyo -n 1)",
+      "continent_code": "$(shuf -e NA EU AS -n 1)",
+      "continent_name": "$(shuf -e 'North America' Europe Asia -n 1)",
+      "country_iso_code": "$(shuf -e US DE JP -n 1)",
+      "country_name": "$(shuf -e 'United States' Germany Japan -n 1)",
+      "location": {"lat": $(echo "scale=6; $RANDOM % 180 - 90" | bc), "lon": $(echo "scale=6; $RANDOM % 360 - 180" | bc)},
+      "name": "geo$((RANDOM % 1000))",
+      "postal_code": "$((10000 + RANDOM % 90000))",
+      "region_iso_code": "region$((RANDOM % 1000))",
+      "region_name": "Region $((RANDOM % 1000))",
+      "timezone": "$(shuf -e PST EST CET JST -n 1)"
+    },
+    "hostname": "host$((RANDOM % 10000))",
+    "id": "hostid$((RANDOM % 10000))",
+    "ip": "$((RANDOM % 256)).$((RANDOM % 256)).$((RANDOM % 256)).$((RANDOM % 256))",
+    "mac": "$(printf '%02x:%02x:%02x:%02x:%02x:%02x' $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256)))",
+    "name": "hostname$((RANDOM % 10000))",
+    "network": {"egress": {"bytes": $((RANDOM % 1000001)), "packets": $((RANDOM % 1000001))}, "ingress": {"bytes": $((RANDOM % 1000001)), "packets": $((RANDOM % 1000001))}},
+    "os": {"family": "$family", "full": "$family $version", "kernel": "kernel$((RANDOM % 1000))", "name": "$family", "platform": "$(shuf -e linux windows macos -n 1)", "type": "$family", "version": "$version"},
+    "pid_ns_ino": "$((1000000 + RANDOM % 9000000))",
+    "risk": {"calculated_level": "$(shuf -e low medium high -n 1)", "calculated_score": $(echo "scale=2; $RANDOM % 100" | bc), "calculated_score_norm": $(echo "scale=2; $RANDOM % 100 / 100" | bc), "static_level": "$(shuf -e low medium high -n 1)", "static_score": $(echo "scale=2; $RANDOM % 100" | bc), "static_score_norm": $(echo "scale=2; $RANDOM % 100 / 100" | bc)},
+    "uptime": $((RANDOM % 1000001))
+  }
+EOF
+    )
+    echo "$host"
+}
+
+# Function to inject documents to agents index
+function index_documents() {
+    local data=$1
+    url="http://$IP:$PORT/$INDEX_NAME/_doc"
+    response=$(curl -s -o /dev/null -w "%{http_code}" -u $USERNAME:$PASSWORD -H 'Content-Type: application/json' -d "$data" -X POST $url)
+    if [[ $response -ne 201 ]]; then
+        echo "Error: $response"
+    fi
+}
+
+function parse_args() {
+    while getopts ":n:o:h" opt; do
+        case ${opt} in
+        h)
+            echo "Usage: $0 [-n <number>]"
+            echo "Options:"
+            echo "  -n <number>  Number of documents to generate. If not provided, the script will prompt for the number of docs to generate."
+            echo "  -h           Display this help message"
+            echo "Example: $0 -n 100"
```

Comment:
-o option is missing

Reply:
Fixed in commit https://github.com/wazuh/wazuh-indexer-plugins/pull/219/commits/ac0741137129fedc04ae0ee99cbc025e03e40bf6